doc/docker-image-install.md

```diff
@@ -0,0 +1,30 @@
+## Using Docker Pull
+
+Pull the latest image from Github Container Registry:
+
+```sh
+docker pull ghcr.io/chrivers/bifrost:master-2025-02-04
```

Comment:
`:master-2025-02-04` may or may not be the latest release right now, but it's not always going to be.

But the `:latest` tag (also the default, if no tag is specified) always points to the latest stable release.

So since we're specifically saying "pull the latest release", we should say

```
docker pull ghcr.io/chrivers/bifrost
```

We could then mention that specific releases are available, with tags such as "master-2025-02-04". By using it as an example of the concept, rather than the example of how to install, it's more clear that there are other versions available :+1:

Reply:
I would suggest making the `:latest` tag explicit rather than implicit.